src/makers/win32/appx.js

```diff
@@ -62,10 +82,15 @@ export default async (dir, appName, targetArch, forgeConfig, packageJSON) => { /
     outputDirectory: outPath,
   });
 
+  if (!opts.publisher) {
+    throw 'Please set config.forge.windowsStoreConfig.publisher or author.name in package.json for the appx target';
+  }
+
   if (!opts.devCert) {
     opts.devCert = await createDefaultCertificate(opts.publisher, outPath);
   }
 
+  // Prepend CN= if the publisher value does not already start with it
   if (!opts.publisher.match(/^CN=/)) {
```

Comment:
Would you like me to make & push these changes? (I think I already gave collaborators write access to the fork for this PR)